dora/core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -617,6 +620,21 @@ public synchronized boolean getUriValidationEnabled() {
     return mUriValidationEnabled;
   }
 
+  /**
+   * Sets current uri path with given uri path.
+   * @param uriPath
+   */
+  public void setUriPath(String uriPath) {
```

Comment:
I'm not a big fan of passing the value through FileSystemContext, I prefer to explicitly pass the value of ufs path

Reply:
I will consider this